.nvmrc

```diff
@@ -0,0 +1 @@
+18.19.0
```

Comment:
need to sync up nvmrc and engine

Reply:
Marking as resolved, with a note here that docker container will be revisited after mvp, we've updated the engine key in package.json to better match our nvmrc.